### PR TITLE
Website: Document more steps for adding maintainers

### PR DESCRIPTION
**Pull request by rata**
This commit adds documentation for the steps we currently know how to
do them and mentions other steps that are needed but we haven't check
the specific steps to do them.
    
Once we go doing the other steps, we will document them here. We just
add them so we have an exhaustive list of all the things to do and don't
forget about them.

### CODE REVIEW

File: website/content/community/maintainers.md [hunk 1]
```diff
@@ -41,16 +41,26 @@ maintainer should object to adding the new maintainer.
 
 When adding a new maintainer, we should:
 
-* Add to CODEOWNERS in the metallb repo.
+* Add to CODEOWNERS in the metallb repo and to the MetalLB github org.
```
Comment (fedepaol) [inline website/content/community/maintainers.md:44]: Can you also mention the metallb-operator ? I think it's mature enough and maintainers should have access there

Comment (rata) [reply website/content/community/maintainers.md:44]: Sure!

**Comment by fedepaol** [comment]
LGTM

**Comment by rata** [comment]
I've just pushed this to v0.12 and updated the live-website branch.

I've also opened https://github.com/metallb/metallb/pull/1263 to fix a minor typo in this PR.

**Comment by rata** [comment]
The website build with this PR is already published :)

**Pull request merged**